File: Rest/app/src/main/java/com/sesong/rest/Activity/LoadingActivity.java
package com.sesong.rest.Activity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.v7.app.AppCompatActivity;

import com.sesong.rest.R;

public class LoadingActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setTheme(R.style.AppTheme_loading);
        setContentView(R.layout.activity_loading);

        Handler handler = new Handler();
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(getBaseContext(), MainActivity.class);  // Intent 선언
                startActivity(intent);   // Intent 시작
                finish();
            }
        }, 2000);  // 로딩화면 시간
    }
}

File: Rest/app/src/main/kotlin/com/sesong/rest/LoadingActivity.kt
package com.sesong.rest

class LoadingActivity : androidx.appcompat.app.AppCompatActivity() {
    override fun onCreate(savedInstanceState: android.os.Bundle?) {
        super.onCreate(savedInstanceState)
        setTheme(com.sesong.rest.R.style.AppTheme_loading)
        setContentView(com.sesong.rest.R.layout.activity_loading)
        android.os.Handler().postDelayed({
            val intent = android.content.Intent(baseContext, MainActivity::class.java) // Intent 선언
            startActivity(intent) // Intent 시작
            finish()
        }, 2000) // 로딩화면 시간
    }
}